packages/api/cms-api/src/dam/files/dto/file.input.ts

```diff
@@ -126,7 +126,7 @@ export class UpdateFileInput {
     @Field(() => ID, { nullable: true })
     @IsUUID()
     @IsOptional()
-    folderId?: string;
+    folderId: string | null | undefined;
```

Comment:
You could use the two new `@IsNullable()` and `@IsUndefinable()` decorators for this field

Reply:
What's the benefit over `@IsOptional()` in this case? Since both, `null` and `undefined`, are valid inputs for the update mutation

Reply:
The code would "show" that you intentionally handle both cases. But the TypeScript type does this as well.

@nsams what do you think?

Reply:
correct: IsOptional is exactly the same as IsNullable and IsUndefinable.

If someone uses IsOptional there can be 2 reasons:

- null and undefinable are allowed
- the dev doesn't know about IsOptional vs IsNullable and IsUndefinable and uses good old IsOptional although it is incorrect

Of course if you look really close you'll notice the typescript types null | undefined...

Maybe we should "deprecate" IsOptional in our code, and event add an eslint rule for that.

Reply:
I changed it to `@IsNullable` and `@IsUndefineable`. I only did it for this property now. Adjusting all properties needs adjustments in fixture generators and services. So I don't want to do it in this PR

Reply:
Before we do that we should add an ESLint rule that marks it as an error